extension/wrapper.cpp

```diff
@@ -116,6 +116,30 @@ Satrec_twoline2rv(PyTypeObject *cls, PyObject *args)
     return (PyObject*) self;
 }
 
+static PyObject *
+Satrec_sgp4init(PyObject *self, PyObject *args)
+{
+    int satnum;
```

Comment:
> satnum gets initialized by the elsetrec struct, so it starts out life as a long int … so it seems like it should always be a long-int.

Yes, I agree entirely! My point is that this line makes it a mere `int`, if I am reading correctly, and should instead read `long satnum`?

Reply:
Fixed with https://github.com/brandon-rhodes/python-sgp4/pull/49/commits/d721c2cee22851ced39502112dfaa016f8e05b28